test/libsolidity/smtCheckerTests/operators/conditional_assignment_always_true.sol

```diff
@@ -0,0 +1,12 @@
+pragma experimental SMTChecker;
+
+contract C {
+	function f(bool b) public pure {
+		require(b);
+		uint c = b ? 5 : 1;
+		assert(c < 5);
+	}
+}
+// ----
+// Warning 6328: (118-131): Assertion violation happens here
+// Warning 6838: (105-106): Condition is always true.
```

Comment:
Shouldn't this be `Condition is always false.`? Since `b` is `true`, `c = 5`

Reply:
It says that condition for `c = b ? 5 : 1` is always `true`, I think it is correct as there is `require(b)` just before condition

Reply:
Ah right, my bad